File: tests/cases/global2.py
#!/usr/bin/env python3


CODE_0 = 0
CODE_1 = 1

CODE_A = "a"
CODE_B = "b"

L_B = {CODE_A}

L_C = {CODE_B: CODE_0}

if __name__ == "__main__":
    assert "a" in L_B
    # Dereferencing items in the dict fails on Rust.
    # Dict compare 'in' does not work on Dart
    # Print of sets and dicts fails on Rust
    print("OK")

File: tests/expected/global2.nim
import sets
import strutils
import tables
let code_0 = 0
let code_1 = 1
let code_a = "a"
let code_b = "b"
let l_b = toHashSet([code_a])
let l_c = {code_b: code_0}.newTable
proc main() =
  assert("a" in l_b)
  echo "OK"

main()
